cypress/integration/homepage/jumbotron_overlay_spec.js

```diff
@@ -0,0 +1,19 @@
+describe('Homepage JumboTron Overlay', function() {
+    before(function() {
+      cy.visit('https://int.crossroads.net');
+    })
+
+    it('Verifies Jumbotron components are only visible when opened', function(){
+      cy.get('[data-automation-id="watch-cta"]').click();
+
+      cy.get('[data-automation-id="jumbotron-overlay"]').should('have.css', 'opacity', '1');
+      cy.get('[data-automation-id="recentService"]').should('have.css', 'opacity', '1');
+      cy.get('[data-automation-id="stream-service"]').should('have.css', 'opacity', '1');
+
+      cy.get('[data-automation-id="overlay-close"]').click();
+
+      cy.get('[data-automation-id="jumbotron-overlay"]').should('have.css', 'opacity', '0');
+      cy.get('[data-automation-id="recentService"]').should('have.css', 'opacity', '1');
+      cy.get('[data-automation-id="stream-service"]').should('have.css', 'opacity', '1');
```

Comment:
These two lines `16` `17` can be omitted. Because they live inside the jumbotron-overlay container they are TECHNICALLY always displayed. Just confirming they have opacity of 1 once is enough.

I realize I'm code reviewing my own test :)

Reply:
:+1: 
I can remove those lines. I just consolidated the steps into one test, didn't really think what should be in it before comitting.